pkg/scheduler/framework/plugins/nodeports/node_ports.go

```diff
@@ -64,7 +64,8 @@ func getContainerPorts(pods ...*v1.Pod) []*v1.ContainerPort {
 		for j := range pod.Spec.Containers {
 			container := &pod.Spec.Containers[j]
 			for k := range container.Ports {
-				ports = append(ports, &container.Ports[k])
+				port := &container.Ports[k]
+				ports = append(ports, port)
```

Comment:
I don't quite think it's more readable - `&container.Ports[k]` is already simple enough.

Reply:
Yeah I feel you're right. Closing